BASE/src/Microsoft.ApplicationInsights/Extensibility/Implementation/Tracing/CoreEventSource.cs

```diff
@@ -9,8 +9,19 @@ internal sealed class CoreEventSource : EventSource
     {
         public static readonly CoreEventSource Log = new CoreEventSource();
 
+#if NETSTANDARD2_0
+        public EventCounter BreezeResponseTimeCounter;
+#endif
+
         private readonly ApplicationNameProvider nameProvider = new ApplicationNameProvider();
 
+#if NETSTANDARD2_0
+        private CoreEventSource()
+        {
+            this.BreezeResponseTimeCounter = new EventCounter("breeze-response-time", this);
```

Comment:
lets rename to "IngestionEndpoint-ResponseTimeMsec"

Reply:
Done.